RetailSuiteAccounting/Operation/08_LockPeriodBookings.md

```diff
@@ -0,0 +1,36 @@
+[!!User Interface lock months](../UserInterface/01_Book.md#lock-months)  
+[!!User Interface bookings](../UserInterface/01a_Bookings.md)  
+
+
+# Lock a period for bookings
+
+The *Lock months* menu entry of the *EDIT* context menu in the *BOOKINGS* tab allows to lock months for bookings.
+
+Once the bookings in a particular booking period are processed, they cannot be changed or deleted any more, and the corresponding booking period is regarded as closed. However, new bookings can be entered in a closed booking period, if necessary. If this is to be avoided, the *Lock months* function can be used to lock one or several periods for bookings. When a booking period is locked, any subsequent transaction will be booked automatically in the following bookable booking period.
+
+In contrast to booking processing, month locking can be undone if necessary, that is, locked months can be made bookable again. For detailed information about processing bookings, see [Process bookings](./07_ProcessBookings.md).
```

Comment:
```suggestion
Once the postings in a particular posting period are processed, they cannot be changed or deleted anymore, and the corresponding postings period is regarded as closed. However, new postings can be entered in a closed posting period, if necessary. To avoid subsequent postings to an already closed posting period, you can lock certain posting periods.   
When a posting period is locked, any subsequent transaction will be automatically posted in the next unlocked posting period.    
In contrast to posting processing, locking a posting period can be undone if necessary.